src/Microsoft.IdentityModel.Tokens/Validation/ValidationParameters.cs

```diff
@@ -112,14 +118,25 @@ public ValidationParameters()
         public AlgorithmValidator AlgorithmValidator { get; set; }
 
         /// <summary>
-        /// Gets or sets a delegate that will be used to validate the audience.
+        /// Allows overriding the delegate that will be used to validate the audience.
         /// </summary>
         /// <remarks>
-        /// If set, this delegate will be called to validate the 'audience', instead of default processing.
+        /// If set, this delegate will be responsible for validating the 'audience', instead of default processing.
         /// This means that no default 'audience' validation will occur.
-        /// Even if <see cref="ValidateAudience"/> is false, this delegate will still be called.
         /// </remarks>
-        public AudienceValidator AudienceValidator { get; set; }
+        /// <exception cref="ArgumentNullException">Thrown when the value is set as null.</exception>
```

Comment:
Same with the wording around exceptions. They are not thrown but returned as part of the result.

Reply:
Will update as part of the validator.